art/estimators/classification/pytorch.py

```diff
@@ -301,6 +301,28 @@ def predict(self, x: np.ndarray, batch_size: int = 128, **kwargs) -> np.ndarray:
 
         return predictions
 
+    def _predict_framework(self, x, **kwargs):
+        """
+        Perform prediction for a batch of inputs.
+
+        :param x: Test set.
+        :type x: `np.ndarray`
+        :param batch_size: Size of batches.
```

Comment:
I think `batch_size` is not necessary here.

Reply:
Ok.